exec/java-exec/src/test/java/org/apache/drill/exec/rpc/security/KerberosHelper.java

```diff
@@ -43,16 +45,20 @@
   public String SERVER_PRINCIPAL;
   private final String testName;
 
-  private File keytabDir;
+  public File keytabDir;
```

Comment:
Generally I am not in favor of public variables (even in test classes helpers). Better to provide getters if needed.